toolstate.toml

```diff
@@ -0,0 +1 @@
+miri = "Testing"
```

Comment:
Can this be located in `src/tools`?

Can this also have a descriptive comment at the top about what's going on here? (possible things to write down, what each state means, etc, etc)